app/views/partials/_footer.haml

```diff
@@ -5,16 +5,6 @@
       = link_to "Neil Johari", "https://github.com/neiljohari/"
       and
       = link_to "Kailan Blanks", "https://github.com/kailan/"
-    %ul.social
-      %li.text-primary
-        %a{:href => "https://twitter.com/SyoStudentGov"}
-          = fa_icon 'twitter'
-      %li
-        %a{:href => "https://www.facebook.com/syosseths"}
-          = fa_icon 'facebook'
-      %li
-        %a{:href => "https://github.com/Syosset/syosset"}
```

Comment:
Could we keep the GitHub link?

Reply:
Sounds good, will have to add the CSS back though. Watch this space.